RPM/scripts/postinst/kaltura-db-config.sh

```diff
@@ -26,6 +26,83 @@ if [ "$#" -lt 4 ];then
 	exit 1
 fi
 
+
+MYSQL_HOST=$1
+MYSQL_SUPER_USER=$2
+MYSQL_SUPER_USER_PASSWD=$3
+MYSQL_PORT=$4
+IS_UPGRADE=$5
+
+#Notice for a non installation
+NO_REMOVAL_NOTICE="This installation has been stopped. In order to try again please tend to stop reason and run '${CYAN}kaltura-config-all.sh${NORMAL}' or '${CYAN}kaltura-db-config.sh${NORMAL}' depending on the initial command which has been used.";
+
+#USed for both connection check, and record check
+EMPTY_HOSTS_RECORDS=`mysql -h$MYSQL_HOST -u$MYSQL_SUPER_USER -p$MYSQL_SUPER_USER_PASSWD -P$MYSQL_PORT -N -e "use mysql; select host,user from mysql.user where user=''"`
```

Comment:
Instead of 'use mysql', you could add mysql as last argument to the mysql command, but since you use mysql.user, you just don't need it.